src/actions/tests/helpers/breakpoints.js

```diff
@@ -1,16 +1,3 @@
-import { makeLocationId } from "../../../utils/breakpoint";
-
-const sourceFixtures = {
```

Comment:
moving this to threadClient.js so that later we can consolidate our threadclient